samples/Azure.ResourceManager.Sample/Generated/Extensions/ArmClientExtensions.cs

```diff
@@ -78,25 +78,25 @@ public static VirtualMachineExtensionImage GetVirtualMachineExtensionImage(this
         }
         #endregion
 
-        #region VirtualMachineExtension
-        /// <summary> Gets an object representing a VirtualMachineExtension along with the instance operations that can be performed on it but with no data. </summary>
+        #region VirtualMachineExtensions
+        /// <summary> Gets an object representing a VirtualMachineExtensions along with the instance operations that can be performed on it but with no data. </summary>
         /// <param name="armClient"> The <see cref="ArmClient" /> instance the method will execute against. </param>
         /// <param name="id"> The resource ID of the resource to get. </param>
-        /// <returns> Returns a <see cref="VirtualMachineExtension" /> object. </returns>
-        public static VirtualMachineExtension GetVirtualMachineExtension(this ArmClient armClient, ResourceIdentifier id)
+        /// <returns> Returns a <see cref="VirtualMachineExtensions" /> object. </returns>
+        public static VirtualMachineExtensions GetVirtualMachineExtensions(this ArmClient armClient, ResourceIdentifier id)
```

Comment:
It looks like the resource is named `VirtualMachineExtensions`. Shall we rename it to ``VirtualMachineExtension` with directive?

Reply:
Is this the case? Let me take a look...

Reply:
This is caused by a bug in `LastWordToSingular`. I have fixed it and added a test case accordingly